miner/util.go

```diff
@@ -0,0 +1,38 @@
+package miner
+
+import (
+	"fmt"
+
+	"golang.org/x/exp/maps"
+
+	"github.com/spacemeshos/go-spacemesh/common/types"
+	"github.com/spacemeshos/go-spacemesh/sql"
+	"github.com/spacemeshos/go-spacemesh/sql/ballots"
+	"github.com/spacemeshos/go-spacemesh/sql/blocks"
+	"github.com/spacemeshos/go-spacemesh/sql/certificates"
+)
+
+func ActiveSetFromBlock(db sql.Executor, epoch types.EpochID) ([]types.ATXID, error) {
```

Comment:
Would it make sense to have a dedicated test for this function? It is indirectly covered by other tests, but maybe crucial enough to have a few test cases directly?

Reply:
https://github.com/spacemeshos/go-spacemesh/issues/4757